tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,686 @@
+{
```

Comment:
<p>determine if you should use it -&gt;</p><p> </p><p>determine if you should use it (including what's supported)</p><p> </p><p>since some people may consider assume that already want to use weight clustering for its compression benefits but have forgotten</p><p>to first consider what's supported (e.g. custom Keras layers). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>